addons/html_builder/static/src/sidebar/option_container.js

```diff
@@ -46,7 +46,11 @@ export class OptionsContainer extends BaseOptionComponent {
     }
 
     get title() {
-        return getSnippetName(this.env.getEditingElement());
+        let title;
+        for (const option of this.props.options) {
```

Comment:
hmm, this is the spec? we take the first title if any. it makes sense, but seems slightly weird